laplace/baselaplace.py

```diff
@@ -628,10 +636,16 @@ def predictive_samples(self, x, pred_type='glm', n_samples=100,
             return self._nn_predictive_samples(x, n_samples)
 
     @torch.enable_grad()
-    def _glm_predictive_distribution(self, X):
+    def _glm_predictive_distribution(self, X, joint=False):
         Js, f_mu = self.backend.jacobians(X)
-        f_var = self.functional_variance(Js)
-        return f_mu.detach(), f_var.detach()
+
+        if joint:
+            f_mu = f_mu.flatten()  # (batch*out)
```

Comment:
Could this be risky when we have multi-output regression or do we not support this anyway?